proto/account/api_key.proto

```diff
@@ -37,6 +37,7 @@ message APIKey {
   string maintainer = 7;
   string api_key = 8;
   string description = 9;
+  string environment_name = 10;
```

Comment:
Please use the `environment_id` instead.
We will need to link it with the project table to get the project name in the future, too.

Reply:
I think we will need `envrionment_name` too, since we will show environment name in new dashboard api keys page.

<img width="1326" alt="image" src="https://github.com/user-attachments/assets/3486f2d4-0f2a-44bf-99bb-0d5ed2256c75">